data/presets/shop/esoteric.json

```
@@ -0,0 +1,16 @@
+{
+    "icon": "maki-shop",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "crystal",
+        "pendant",
+        "essential oil"
```

```suggestion
    "terms": [
        "crystal",
        "essential oil",
        "pendant",
        "woo"
```

sorts terms, adds "woo" as a term (idea from https://community.openstreetmap.org/t/is-there-anything-wrong-with-shop-esoteric-tagging/137372/2 )

psychic, alternative, new age, acupuncture, aromatherapy, meditation are other possible terms from https://github.com/SomeoneElseOSM/SomeoneElse-style/blob/master/shared_lua.lua#L2453